Source/diablo.cpp

```diff
@@ -1424,73 +1413,124 @@ void InitKeymapActions()
 	    N_("Open Speedbook."),
 	    'S',
 	    DisplaySpellsKeyPressed,
-	    [&]() { return !IsPlayerDead(); });
+	    nullptr,
+	    CanPlayerTakeAction);
 	sgOptions.Keymapper.AddAction(
 	    "QuickSave",
 	    N_("Quick save"),
 	    N_("Saves the game."),
 	    DVL_VK_F2,
 	    [] { gamemenu_save_game(false); },
-	    [&]() { return !gbIsMultiplayer && !IsPlayerDead(); });
+	    nullptr,
+	    [&]() { return !gbIsMultiplayer && CanPlayerTakeAction(); });

```

Comment:
is there an issue with letting the player save when the game i paused?

Reply:
It isn't allowed in master and I didn't changed it.
But i tested it, when you save the game it gets unpaused. But else it seem to work.
Related note: you currently can't load a game when the game is paused (in master and this pr).
Thoughts?

Reply:
I think we will keep it the way you did it here. I could see wanting to save when it's paused, but I also feel that it shouldn't be possible to load, and that just seams inconsistent on my part :)